cmd/bucket-handlers.go

```diff
@@ -291,6 +291,14 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 		return
 	}
 
+	// Deny if WORM is enabled
+	if globalWORMEnabled {
+		// Not required to check whether given objects exist or not, because
+		// DeleteMultipleObject is always successful irrespective of object existence.
+		writeErrorResponse(w, ErrMethodNotAllowed, r.URL)
```

Comment:
we should write success response, i.e status code must be 200 and the response body should contain the error XML with error code, message etc.
See Example 3 response in https://docs.aws.amazon.com/AmazonS3/latest/API/multiobjectdeleteapi.html.

Reply:
IMO this is not like `AccessDenied` or any other S3 type error hence `405 Method not allowed`.  This method on the resource is not allowed.

If this point is not valid, we would need to discuss and fix accordingly.